extensions/azurecore/package.json

```diff
@@ -283,6 +283,11 @@
           "when": "viewItem == azure.resource.itemType.azureDataExplorer",
           "group": "inline"
         },
+        {
+          "command": "azure.resource.connectsqlserver",
+          "when": "viewItem == azure.resource.itemType.azureMonitor",
```

Comment:
It would be preferable not to have azurecore need to know about Azure Monitor (or any other extensions). Is there a reason why you couldn't have this command contributed from the azure monitor extension instead? Generally all contributions related to a particular feature should be done from the extension for that feature.

Reply:
I think the azurecore has to know about to some level of the datasources it's rendering in the azurewidget. Besides this config, I also have to include the datasource icons, and a specific provider, service, and TreeDataProvider.

Reply:
I sent a message to @alanrenmsft with a link to this convo.